drivers/SPI.h

```diff
@@ -143,6 +143,14 @@ class SPI : private NonCopyable<SPI> {
      */
     virtual void unlock(void);
 
+    /** SPI block read dummy data
+      * SPI requires master to send dummy data, in order to perform read operation.
```

Comment:
Edits in bold:
SPI requires **the** master to send dummy data **during** **a** read operation.
 